src/sqlfluff/api/simple.py

```diff
@@ -123,8 +123,17 @@ def fix(
     linter = Linter(config=cfg)
 
     result = linter.lint_string_wrapped(sql, fix=True)
-    fixed_string = result.paths[0].files[0].fix_string()[0]
-    return fixed_string
+    fix_even_unparsable = cfg.get("fix_even_unparsable")
```

Comment:
For API usage, I think this should be an available kwarg on `fix()`, and not just accessible via the config.

Reply:
Done.